From 35acf3d707f9583498d371bd85c9e089d860d20d Mon Sep 17 00:00:00 2001
From: xSlazharx <creamylofly@gmail.com>
Date: Wed, 7 May 2025 14:41:51 +0000
Subject: [PATCH 3/3] selinux: hooks: Remove `static inline` at `current_sid`

Signed-off-by: xSlazharx <creamylofly@gmail.com>
---
 security/selinux/hooks.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/security/selinux/hooks.c b/security/selinux/hooks.c
index 95ae193d79f1..c27400af6973 100644
--- a/security/selinux/hooks.c
+++ b/security/selinux/hooks.c
@@ -214,7 +214,7 @@ static inline u32 task_sid(const struct task_struct *task)
 /*
  * get the subjective security ID of the current task
  */
-static inline u32 current_sid(void)
+u32 current_sid(void)
 {
 	const struct task_security_struct *tsec = current_security();
 
-- 
2.49.0

